src/mpi/romio/adio/ad_lustre/ad_lustre_open.c

```diff
@@ -57,6 +57,7 @@ void ADIOI_LUSTRE_Open(ADIO_File fd, int *error_code)
     lum = (struct lov_user_md *)ADIOI_Calloc(1,lumlen);
 
      value = (char *) ADIOI_Malloc((MPI_MAX_INFO_VAL+1)*sizeof(char));
+    int value_sz = (MPI_MAX_INFO_VAL+1)*sizeof(char);
```

Comment:
can you move this variable declaration up with the others, and also make it a `size_t`? Then we can avoid doing the size calculation here and also in `ADIOI_Malloc`